test/e2e_tests/login.spec.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Wire
+ * Copyright (C) 2025 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {test, expect} from '@playwright/test';
+
+import {createPersonalUserViaBackdoor, deleteUser} from './backend/backend';
+import {ClientUser} from './backend/clientUser';
+import {DataShareConsentModal} from './pages/dataShareConsentModal.page';
+import {LoginPage} from './pages/login.page';
+import {WelcomePage} from './pages/welcome.page';
+import {getCredentials} from './utils/credentialsReader';
+
+const onePasswordItemName = 'BackendConnection staging-with-webapp-master';
+const webAppPath = getCredentials(onePasswordItemName, 'webappUrl');
+const createdUsers: ClientUser[] = [];
+
+test('Verify sign in error appearance in case of wrong credentials', {tag: ['@TC-3465', '@smoke']}, async ({page}) => {
+  const incorrectEmail = 'blablabla@gmail.com';
```

Comment:
```suggestion
  const incorrectEmail = 'blablabla@wire.engineering';
```

Nitpick: Instead of gmail can we use wire.engineering?